src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,94 @@
+package wolox.training.models;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.repositories.BookRepository;
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
+
+@Entity
+public class User {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    private  String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @Column(nullable = false)
+    @ManyToMany(cascade = { CascadeType.MERGE, CascadeType.REFRESH})
+    @JoinTable(name = "book_user",
+            joinColumns = @JoinColumn(name = "book_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id",
+                    referencedColumnName = "id"))
+    private List<Book> books;
+
+    public User(){ }
+
+    public User(String username, String name, LocalDate birthdate) {
+        this.username = username;
+        this.name = name;
+        this.birthdate = birthdate;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
```

Comment:
This might not be necessary as the id gets set by JPA/Hibernate, just bear it in mind.